Tests/JWSTests.swift

```diff
@@ -54,13 +65,17 @@ class JWSTests: CryptoTestCase {
     // MARK: - RSA Tests
 
     @available(*, deprecated)
-    private func performTestRSASign(algorithm: SignatureAlgorithm, compactSerializedJWS: String) {
```

Comment:
I decided to delete this argument `compactSerializedJWS: String` because it was not used in the function at all.